daemon/state.go

```diff
@@ -42,6 +44,14 @@ func (s *State) String() string {
 		return fmt.Sprintf("Up %s", units.HumanDuration(time.Now().UTC().Sub(s.StartedAt)))
 	}
 
+	if s.removalInProgress {
+		return "Removal In Progress"
+	}
+
+	if s.Dead {
```

Comment:
But that's a separate call in separate function. First call is in String() and second call is in StateString().  So does not look like there is redundancy.